code/modules/projectiles/guns/rifles.dm

```diff
@@ -164,12 +164,13 @@
 		/obj/item/attachable/attached_gun/flamer/advanced,
 		/obj/item/attachable/attached_gun/grenade,
 		/obj/item/attachable/scope/mini/nsg23,
+		/obj/item/attachable/attached_gun/shotgun/af13,
```

Comment:
Are you forgetting to add `/obj/item/attachable/suppressor/nsg,` as one of the allowed attachments?